src/jitter.rs

```diff
@@ -424,18 +422,28 @@ impl JitterRng {
 
     fn gen_entropy(&mut self) -> u64 {
         trace!("JitterRng: collecting entropy");
-        
-        // Prime `self.prev_time`, and run the noice sources to make sure the
+
+        // Prime `ec.prev_time`, and run the noice sources to make sure the
         // first loop round collects the expected entropy.
-        let _ = self.measure_jitter();
+        let mut ec = EcState {
+            prev_time: self.timer.get_nstime(),
+            last_delta: 0,
+            last_delta2: 0,
+            mem: [0; MEMORY_SIZE],
```

Comment:
If this is statically initialised to zero each time, what's to stop the compiler optimising the memory access out? Even if it doesn't, this will be in the lowest-level cache it fits in already.

Reply:
> If this is statically initialised to zero each time, what's to stop the compiler optimising the memory access out?

The `black_box(ec.mem[0]);` ~sprinkled around~ placed strategically.

> Even if it doesn't, this will be in the lowest-level cache it fits in already.

Good point, I will do some measurements.

Reply:
I collected measurements with `timer_stats` for 100_000 rounds, with the CPU timer source in `timer_stats` disabled. So I only measured the `mem_access` noise source.

The result of the NIST Entropy Estimation test on the current version is: 0.449745 (on my PC).
And with new version it is: 0.465139. So no difference.